pippy/PipelineDriver.py

```diff
@@ -1094,6 +1126,8 @@ def __init__(self, remote_stage_executor_rrefs, stage_to_executor, module, cur_m
         self.args_iter = iter(self.args)
         self.batch_id = batch_id
         self.num_microbatches = num_microbatches
+        # Dict from stage id to a list holding the coalesced getitem indices
+        self.stage_indices : Dict[int, List[Tuple]] = {}
```

Comment:
nit: maybe make this more specific like `stage_output_indices`

Also same comment as above about more precise type annotation

Reply:
Thanks, fixed.